inc/namespace.php

```diff
@@ -230,6 +291,22 @@ function override_yoast_seo_options( $options ) : ?array {
 	return $options;
 }
 
+/**
+ * Render a notice on the Yoast SEO Dashboard page that social settings have been configured in the Altis config.
+ */
+function social_options_overridden_notice() {
+	$screen = get_current_screen();
+
+	if ( $screen->base !== 'toplevel_page_wpseo_dashboard' ) {
+		return;
+	}
+
+	$classes = 'notice notice-warning is-dismissable';
+	$message = __( 'Social metadata has been saved in the Altis configuration file. Use the Altis configuration file to make changes to the Social SEO settings.', 'altis-seo' );
```

Comment:
Could maybe provide a link to the docs on this 🤔  Would need to extract the version and stuff to get the right link though so maybe not worth it right now.

Also maybe reword the language around "metadata saved in the config file", perhaps just "set", minor thing it just made me pause. Because the call to action "Use the config file to make changes" is written there that might be confusing for non developers or people using Altis who aren't aware of the technical components.

I would be tempted to just drop this notice altogether, or remove the second sentence or make it clear who's responsibility it would be to make changes. Maybe like:

```suggestion
	$message = __( 'Social media profile data has been set in the Altis configuration file and cannot be modified here.', 'altis-seo' );
```

Reply:
Updating this a bit to only display once in addition to changing the language.